### DISCUSSION TITLE
Submenu for sidenav

### DISCUSSION THREAD

**Discussion by AlexandruAndrei3** [frangoteam/FUXA#312] [Ideas]
Hi,
When adding multiple views and tabs the home side navigation tends to get pretty crowded. For a better structure I would suggest that it would be useful to offer the user the option to add submenu items in the layout settings something similar to a dropdown but with the same style set up in the Navigation Side Menu Type. 
![Screenshot 2022-03-28 113442](https://user-images.githubusercontent.com/94168341/160359622-f3dc674c-2a78-48b7-8631-340fdaa7402a.png)

Thanks, for your time!

**Comment by unocelli** [comment]
Hi,
It is an interesting feature but I would give it a medium priority. :wink:

**Comment by liulujian00** [comment]
I think this is a good design for the interface layout. I am currently encountering the problem of insufficient tab classification. This submenu can just solve my problem. I would like to ask, how is the development of this function and when will it be released?

**Reply by unocelli** [reply]
Hi, there is already a issue for this feature #962
